cli-dotnet.test.js

```diff
@@ -0,0 +1,98 @@
+// -----------------------------------------------------------------------------------------
+// #region Imports
+// -----------------------------------------------------------------------------------------
+
+const {
+    ERROR_OUTPUT_STRING,
+    HELP_OPTIONS
+} = require("./_modules/constants");
+const testUtils = require("./tests/test-utils");
+const child_process = require("child_process");
+
+// #endregion Imports
+
+// -----------------------------------------------------------------------------------------
+// #region Tests
+// -----------------------------------------------------------------------------------------
+
+describe("cli-dotnet", () => {
+    let _tmpDir;
+    let _cleanupTmpDir = () => { };
+    beforeEach(() => {
+        // Verify that dotnet is installed before continuing.
+        testUtils.verifyDotnetPath();
+
+        // Before each test, create a temporary directory for the test to work with. We can muck around
+        // for lifetime of the test and it will be cleaned up in the 'afterEach' hook.
+        const { tmpDir, cleanupTmpDir } = testUtils.createAndUseTmpDir("cli-dotnet");
+        _tmpDir = tmpDir;
+        _cleanupTmpDir = cleanupTmpDir;
+    });
+
+    afterEach(() => {
+        // Remove the temporary directory and return to whence we came.
+        _cleanupTmpDir();
+    });
+
+    // -----------------------------------------------------------------------------------------
+    // #region -b, --build
+    // -----------------------------------------------------------------------------------------
+
+    describe("-b, --build", () => {
```

Comment:
Thought in seeing these flags again. Almost feeling like a command could have a typed object or _something_ like 'flags'. 

```
flags: {
    buildShort: "-b",
    build: "--build"
}
```

Reply:
I opted for an array of options to be exported from the module itself - hope that works too! I'm not sure when we would need to reference any of the individual flags directly, so I figured this format makes more sense for referencing in tests, or when registering the module in parent-level command.